File: MODUL 6/SOAL 3/PRAK503-2210817210026-Zulfa Auliya Akbar.py
def maksimal (a,b):
    return max(a,b)
def minimal (a,b):
    return min(a,b)


batas = 0
maks = -100000
minim = 100000
bilangan = int(input())
while(batas<bilangan):
    nilai = map(int,input().split())
    for i in nilai :
        maks   = maksimal(maks, i)
        minim = minimal(minim, i)
        batas+=1
print(maks,minim)

File: MODUL 6/SOAL 3/PRAK503-2210817210026-Zulfa Auliya Akbar.c
#include <stdio.h>
#define MAX(x,y) (((x) > (y)) ? x : y)
#define MIN(x,y) (((x) < (y)) ? x : y)
int maksimal(int a, int b){

return MAX(a,b);
}
int minimal(int a, int b){

return MIN(a,b);
}
int main(){
int batas = 0;
int maks = -100000;
int minim = 100000;
int bilangan;
scanf("%d", &bilangan);

while(batas < bilangan){
int nilai;
scanf("%d", &nilai);
maks = maksimal(maks, nilai);
minim = minimal(minim, nilai);
batas++;
}

printf("%d %d",maks,minim);
}